src/components/DomainFilter/DomainFilter.container.ts

```diff
@@ -7,4 +7,12 @@ const mapStateToProps = (state: AppState) => ({
   domains: getDomains(state)
 })
 
-export default connect(mapStateToProps)(DomainFilter)
+
+type OwnProps = {
+  value: FilterValue;
+  onValueChange?: (value: FilterValue) => void;
+}
+
+
+// @ts-ignore todo(@alexstrat): fix connect typing for outter props
```

Comment:
I remember that typing correctly `OwnProps` (external props) with `connect` (via HOC) was challenging. To correctly fix this I would consider switching this component to React hooks that is more flexible and explicit with typing than HOC.